lib/stacks/api/restApi.ts

```diff
@@ -305,7 +306,8 @@ export class RestApiStack extends Stack {
             ENABLE_GROUNDTRUTH: ENABLE_GROUNDTRUTH,
             SYSTEM_BANNER: props.systemBannerConfiguration,
             APPLICATION_NAME: APPLICATION_NAME,
-            DATASET_BUCKET: props.dataBucketName
+            DATASET_BUCKET: props.dataBucketName,
+            AWS_REGION,
```

Comment:
should this be

```suggestion
AWS_REGION: AWS_REGION,
```

this?

Reply:
Not totally necessary in this case. If the property name and the variable you're assigning are the same you can omit the explicit assignment. Admittedly this isn't totally consistent across the codebase.

Reply:
We aren't consistent about this throughout the project and I tend to use it when I can but I'm going to accept this suggestion to keep consistency in this file.